ineffassign.go

```diff
@@ -35,7 +35,11 @@ func main() {
 		}
 		fmt.Fprintln(os.Stderr, "\nFlags:")
 		flag.PrintDefaults()
+		ineffassign.Analyzer.Flags.PrintDefaults()
```

Comment:
Won't this print the analyzer flags a second time, since they are all added to the global flags with your addition below?

Reply:
Good catch